service-front/Dockerfile

```diff
@@ -1,7 +1,72 @@
+FROM node:21.1.0-alpine3.17 as asset-env
```

Comment:
The PHP FPM container shouldn't be serving the assets, you want this to build into the nginx container so static files get served by the web server before you hit the PHP parser..